conf/salt/postgres.sls

```diff
@@ -6,9 +6,9 @@
 postgres_repo:
   pkgrepo.managed:
     - humanname: Postgres PPA
-    - name: deb http://apt.postgresql.org/pub/repos/apt/ {{ grains['lsb_distrib_codename'] }}-pgdg main
-    - file: /etc/apt/sources.list.d/postgres.list
-    - key_url: salt://project/pgdg.pub
```

Comment:
On Thu, Oct 22, 2015 at 12:48 PM, Vinod Kurup <notifications@github.com>
wrote:

> How was it failing before? Could it be because this file was missing in
> the salt tree?


​Maybe. It was one of those useless error messages, something like "type
boolean is not iterable" and that was all.